src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -226,7 +226,7 @@ def add_electrodes(recording: BaseRecording, nwbfile: pynwb.NWBFile, metadata: d
 
     assert all(
         [property["name"] != "group" for property in electrodes_metadata]
-    ), "Passing metadata field 'group' is deprecated; pass group_name instead!"
+    ), "Passing metadata field 'group' is deprecated; pass group_name instead!"  # TODO: remove completely by 8/21/2024
```

Comment:
What would "removing this completly" would mean?

Reply:
Oh, I thought that was related to the recent changes related to `group` for recording objects - looks like that one's been effectively hard-deprecated for a while now